pkgs/development/python-modules/jedi-language-server/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, poetry
+, docstring-to-markdown
+, jedi
+, pygls
+, pytestCheckHook
+, pyhamcrest
+, python-jsonrpc-server
+}:
+
+buildPythonPackage rec {
+  pname = "jedi-language-server";
+  version = "0.34.8";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "pappasam";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-mJGgDDjPZXde4M4OHwj81KYoaFXFAwOZ+v18YE+arFE=";
+  };
+
+  nativeBuildInputs = [
+    poetry
```

Comment:
Can we use poetry-core?

Reply:
> Can we use poetry-core?

No:

```
python3.9-jedi-language-server> ModuleNotFoundError: No module named 'poetry.masonry'
```

Reply:
Then we would need patching. If you don't feel comfortable doing that it is not a problem.